### PR TITLE
fix various typos in eventing controller

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- fix typos in comments and code, as well as minor formatting issues
- rename function and fix various spelling mistakes

**Related issue(s)**

### CODE REVIEW

**Review by grego952 — APPROVED** [review]

File: components/eventing-controller/config/default/manager_auth_proxy_patch.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This patch inject a sidecar container which is a HTTP proxy for the
+# This patch injects a sidecar container which is an HTTP proxy for the
 # controller manager, it performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
```
Comment (FriedrichWilken) [inline components/eventing-controller/config/default/manager_auth_proxy_patch.yaml:2]:
```suggestion
# This patch injects a sidecar container which is an HTTP proxy for the controller manager
# which performs RBAC authorization against the Kubernetes API using SubjectAccessReviews.
```

File: components/eventing-controller/controllers/subscription/beb/reconciler.go [hunk 2]
```diff
@@ -98,8 +98,9 @@ func NewReconciler(ctx context.Context, client client.Client, applicationLister
 // +kubebuilder:rbac:groups=gateway.kyma-project.io,resources=apirules,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:printcolumn:name="Ready",type=bool,JSONPath=`.status.Ready`
 
-// TODO: Optimize number of reconciliation calls in eventing-controller #9766: https://github.com/kyma-project/kyma/issues/9766
+// Reconcile sets up
```
Comment (FriedrichWilken) [inline components/eventing-controller/controllers/subscription/beb/reconciler.go:101]: ?

**Review by nachtmaar — COMMENTED** [review]
Left some nits. But I can approve it anytime. Let me know.

File: components/eventing-controller/controllers/subscription/nats/reconciler.go [hunk 3]
```diff
@@ -59,16 +59,14 @@ var (
 )
 
 const (
-	NATSFirstInstanceName = "eventing-nats-1" // NATSFirstInstanceName the name of first instance of NATS cluster
-	NATSNamespace         = "kyma-system"     // NATSNamespace namespace of NATS cluster
+	FirstInstanceName     = "eventing-nats-1" // FirstInstanceName the name of first instance of NATS cluster
```
[the remark this answers is not in the document]
Comment (k15r) [reply components/eventing-controller/controllers/subscription/nats/reconciler.go:62]: Good catch, lets unexport it and keep the name

File: components/eventing-controller/pkg/handlers/nats_test.go [hunk 4]
```diff
@@ -1014,7 +1014,7 @@ func TestSubscription_NATSServerRestart(t *testing.T) {
 	natsServer, natsPort := startNATSServer()
 	defer natsServer.Shutdown()
 	defaultLogger := getLogger(g, kymalogger.INFO)
-	// The reconnect configs should be large enough to cover the NATS server restart period
+	// The `reconnects` configs should be large enough to cover the NATS server restart period
```
[the remark this answers is not in the document]
Comment (k15r) [reply components/eventing-controller/pkg/handlers/nats_test.go:1017]: the original text felt a bit awkward to read. Any other idea how to improve readability ?

**Review by grego952 — APPROVED** [review]

**Review by nachtmaar — APPROVED** [review]

**Pull request merged**